apps/src/lab2/views/components/TextToSpeech.tsx

```diff
@@ -10,6 +10,7 @@ import moduleStyles from './TextToSpeech.module.scss';
 
 interface TextToSpeechProps {
   text: string;
+  higher?: boolean;
```

Comment:
nit: should we rename this prop to smth like `isPositionedOnTop`? It feels like for `higher` prop might be a little harder to understand quickly what it's responsible for

Reply:
Good call; fortunately we no longer need it.